services/minio/.env

```diff
@@ -0,0 +1,6 @@
+MINIO_ACCESS_KEY=12345678
```

Comment:
rename as `.env-defaults` and autogenerate `.env` if it does not exist (as in simcore Makefile)?

Reply:
actually was wondering why we have it like this?

Reply:
will be fixed in next PR